sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -33,27 +34,6 @@ import org.apache.spark.sql.catalyst.util.DateTimeUtils
 
 /**
  * :: Experimental ::
- * Information about updates made to stateful operators in a [[StreamingQuery]] during a trigger.
```

Comment:
I moved this below to keep the StreamingQueryProgress class first in the file. More important code first.